src/javascript/JContent/ContentNavigation/ContentNavigation.jsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {Accordion, SecondaryNav} from '@jahia/moonstone';
 import {useTranslation} from 'react-i18next';
-import NavigationHeader from './NavigationHeader';
 
-const ContentNavigation = ({accordionItems, mode, siteKey, handleNavigation}) => {
-    const {t} = useTranslation('jcontent');
+const ContentNavigation = ({accordionItems, mode, siteKey, handleNavigation, header}) => {
+    const {t} = useTranslation();
```

Comment:
you can probably leave 'jcontent' as default namespace, that won't have any impact if a ns is specified in the label